nebula3/data/DataObject.py

```diff
@@ -5,7 +5,7 @@
 #
 # This source code is licensed under Apache 2.0 License.
 
-
+from typing import Dict, List, Set
```

Comment:
@Aiee are data/*.py hand coded or generated by thrift?

Reply:
These files are manually added. All generated files have a header saying `generated by thrift`

Reply:
Got it, then it's good to be modified :)